source/options.css

```diff
@@ -110,7 +110,8 @@ h2 ~ h2 {
 	text-decoration: line-through;
 }
 
-.js-features :not(:checked) + .info .extended-options {
+.js-features :not(:checked) + .info .extended-options,
+.feature[hidden] {
```

Comment:
This shouldn't be needed, `[hidden] {display:none}` is offered by the browser

Reply:
It doesn't seem to be set... having the hidden attribute was not hiding the entries.